packages/braid-design-system/lib/css/globalTypographyStyles.ts

```diff
@@ -0,0 +1,52 @@
+import type { StyleRule } from '@vanilla-extract/css';
+import { vars } from '../../lib/themes/vars.css';
+import { responsiveStyle } from './responsiveStyle';
+import * as typographyStyles from '../css/typography.css';
+
+interface GlobalTextStyleProps {
+  weight?: keyof typeof typographyStyles.fontWeight;
+  size?: keyof typeof typographyStyles.textSizeUntrimmed;
+}
+
+export const globalTextStyle = ({
+  weight = 'regular',
+  size = 'standard',
+}: GlobalTextStyleProps = {}): StyleRule => ({
+  fontFamily: vars.fontFamily,
+  fontWeight: vars.textWeight[weight],
```

Comment:
Were these supposed to be `typographyStyles.fontWeight` and `typographyStyles.fontWeight[weight]`?

Reply:
Deliberately not using sharing the internal typography classes here. Providing this to consumers as a data structure (StyleRule) allows consumers to control the selector, e.g.:

```ts
globalStyle(`${scope} p`, globalTextStyle(...));
```

Reply:
Gotcha, I saw the style rules looked like the ones in in `typographyStyles.fontWeight` and `typographyStyles.textSizeUntrimmed`.

`GlobalTextStyleProps` also references `keyof typeof typographyStyles.*` and the values are not used here, so to me it looked like there is an invisible coupling between the two (`globalTextStyle` and `typographyStyles`).

Reply:
Ah now that I can update, they should be `keyof typeof vars.textWeight` etc. Updating...